centreon/packages/ui/src/Icon/DeleteIcon.tsx

```diff
@@ -0,0 +1,7 @@
+import { JSX } from 'react';
+
+import { DeleteOutline } from '@mui/icons-material';
+
+export const DeleteIcon = (): JSX.Element => {
+  return <DeleteOutline className="text-error-main text-[20px]" />;
```

Comment:
Why did you set the font size to 20px? 24px is the standard size

Reply:
I only extracted the Icon on a separated component, nothing changes in term of size, color...